core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1065,7 +1065,7 @@ object KafkaConfig {
       .define(AdvertisedListenersProp, STRING, null, HIGH, AdvertisedListenersDoc)
       .define(ListenerSecurityProtocolMapProp, STRING, Defaults.ListenerSecurityProtocolMap, LOW, ListenerSecurityProtocolMapDoc)
       .define(ControlPlaneListenerNameProp, STRING, null, HIGH, controlPlaneListenerNameDoc)
-      .define(ControllerListenersProp, LIST, null, HIGH, ControllerListenersDoc)
+      .define(ControllerListenersProp, STRING, null, HIGH, ControllerListenersDoc)
```

Comment:
what's the reasoning for making this a STRING instead of a LIST, given that it can have multiple values?

Reply:
I was looking at the way CSV lists are parsed (other listener props) and they are all accepted as a string and then tokenized into a list

Reply:
IOW all the utility functions for listeners treat the listeners as a string. So, I figured it makes sense to keep it that way for ControllerListeners too

Reply:
ok, we can do this to be consistent with the existing code.